ui/app/templates/components/transform-role-edit.hbs

```diff
@@ -58,7 +58,11 @@
       <MessageError @model={{this.model}} />
       <NamespaceReminder @mode={{this.mode}} @noun="Transform role" />
       {{#each this.model.attrs as |attr|}}
-        <FormField data-test-field @attr={{attr}} @model={{this.model}} />
+        {{#if attr.options.readOnly}}
+          <ReadonlyFormField @attr={{attr}} @value={{get this.model attr.name}} />
```

Comment:
Caught this in testing -- we were allowing users to update the name field on edit